frontend/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": false,
```

Comment:
false er vel default for `singleQuote`?

Reply:
Riktig. Men sida jeg har den satt til `true` i egen global config (fordi jeg foretrekker single quotes), har jeg eksplisitt satt den til false her for å override.